SlopCrew.Server/GitHubDownloader.cs

```diff
@@ -0,0 +1,54 @@
+using Serilog;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace SlopCrew.Server {
+    internal class GitHubDownloader {
+        public async Task<Dictionary<string, string>> DownloadFilesFromDirectory(string repositoryOwner, string repositoryName, string directoryPath = "") {
+            var files = new Dictionary<string, string>();
+
+            using var client = new HttpClient();
+
+            var apiUrl = $"https://api.github.com/repos/{repositoryOwner}/{repositoryName}/contents/{directoryPath}?ref=main";
```

Comment:
I'd prefer to submodule the race config into the repository, falling back to a user-specified config dir, but this is fine for now.